app/src/main/java/co/develhope/meteoapp/today/TodayScreenFragment.kt

```diff
@@ -0,0 +1,98 @@
+package co.develhope.meteoapp.today
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import co.develhope.meteoapp.R
+import co.develhope.meteoapp.databinding.FragmentTodayScreenBinding
+import co.develhope.meteoapp.today.adapter.TodayAdapter
+import org.threeten.bp.LocalTime
+import org.threeten.bp.OffsetTime
+import org.threeten.bp.ZoneOffset
+import org.threeten.bp.format.DateTimeFormatter
+
+class TodayScreenFragment : Fragment() {
+    private var _binding: FragmentTodayScreenBinding? = null
+    private val binding get() = _binding!!
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = FragmentTodayScreenBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
```

Comment:
Non trovo dove "leggi" idati ma solo dove crei la lista che rappresenta la ui

Reply:
Momentaneamente non avendo una API da dove prendere i dati ho creato solamente una mock-list, quindi in modo effettivo non leggo dati ma li creo e aggiungo in una lista che poi verrà messa dentro alla RecyclerView

Reply:
è quello il problema. devi leggerli da una classe dove li crei mock, perchè al fragment non interessa come vengono generati/letti deve avere solo un punto di accesso per leggerli. 
Così quando avrai le api, che sostiuirai alla lista mock , nel fragment non cambierà niente.